src/store/ducks/transaction/actionTransactions.ts

```diff
@@ -3,9 +3,12 @@ import { createSlice, PayloadAction, createSelector } from '@reduxjs/toolkit';
 import { RootState } from 'store/types';
 import snxJSConnector from 'utils/snxJSConnector';
 import { fetchDelegateWalletInfoRequest } from '../delegates/delegateWalletInfo';
+import orderBy from 'lodash/orderBy';
 
 export interface Transaction {
```

Comment:
happy to switch to `type` instead of `interface` (can definitely be an incremental change)